docs/running-multi-parsers.rst

```diff
@@ -284,6 +284,8 @@ Registering a ``subparser var`` to a base parser is enough for the
 bottom up choice. For handling the top down choice (e.g. specifying
 ``--language-force=<subparser>`` in a command line), more code is needed.
 
+.. TODO: Q to @masatake-san: I cannot parse the following paragraph.
+
```

Comment:
Me, too...

In the top down choice, the subparser must call ``scheduleRunningBasepaser`` in its ``parser`` method.
Here, ``parser`` method means a function assigned to the ``parser`` member of the `parserDefinition`` of the subparser.
``scheduleRunningBaseparser`` is declared in ``main/subparser.h``.

Reply:
Now I understand.  I will fix it.